src/sensor/ping360.h

```diff
@@ -358,12 +358,15 @@ class Ping360 : public PingSensor
     static const uint16_t _firmwareDefaultTransmitFrequency = 740;
     static const uint16_t _firmwareDefaultNumberOfSamples = 1024;
 
+    // The default transmit frequency to operate with
+    static const uint16_t _viewerDefaultTransmitFrequency = 750;
```

Comment:
what is the difference between this variable and _firmwareDefaultTransmitFrequency ?

Reply:
The firmware uses 740 khz by default.

Reply:
My points is, where `_firmwareDefaultTransmitFrequency` is being used ?

Reply:
It is not, but it is a useful constant to have for reference.